### PR TITLE
Feature/dont look at ball in ball handle

### PR DESCRIPTION
**Pull request by mrlukasbos**

### CODE REVIEW

File: roboteam_ai/src/world/Field.cpp [hunk 1]
```diff
@@ -38,23 +38,20 @@ Vector2 Field::get_their_goal_center() {
...
-    float halfLength = _field.field_length*0.5f;
-    float halfWidth = _field.field_width*0.5f;
+bool Field::pointIsInField(const Vector2 &point, double margin) {
+    auto field = get_field();
 
-    return (point.x <= halfLength - margin &&
-            point.x >= - halfLength + margin &&
-            point.y <= halfWidth - margin &&
-            point.y >= - halfWidth + margin);
+    double halfLength = field.field_length*0.5f - margin;
```
Comment (mrlukasbos) [inline roboteam_ai/src/world/Field.cpp]: the margin is now divided by two since it is only applied once! This thus looks misleading... The test does fail as well on the (literal) edge cases.

**Comment by mrlukasbos** [comment]
@ThijsLuttikhuis what does your latest commit do? It breaks the tests while we should've actually merged this PR

File: roboteam_ai/src/control/ballHandling/DribbleBackwards.cpp [hunk 2]
```diff
@@ -166,40 +166,58 @@ RobotCommand DribbleBackwards::sendTurnCommand() {
...
-    command.dribbler = 28;
+    command.dribbler = 31;
     command.angle = lockedAngle;
     command.vel = lockedAngle.toVector2(- maxVel);
 
+
     // check if the robot is still on the virtual line from ball->pos to the target
     if (control::ControlUtils::distanceToLine(robot->pos,
-            backwardsDribbleLine.first, backwardsDribbleLine.second) > errorMargin*2.5) {
+            backwardsDribbleLine.first, backwardsDribbleLine.second) > errorMargin*5) {
```
Comment (mrlukasbos) [inline roboteam_ai/src/control/ballHandling/DribbleBackwards.cpp]: 5.0
